package.json

```diff
@@ -2,7 +2,7 @@
   "name": "BitGoWalletRecoveryWizard",
   "author": "BitGo, Inc.",
   "description": "A UI-based desktop app for BitGo Recoveries",
-  "version": "1.8.0",
+  "version": "1.8.2",
```

Comment:
Double patch version bump is a bit strange. Probably should be a new minor version (this is a new feature).